packages/website/blog/table_v2/table-v2-release.mdx

```diff
@@ -0,0 +1,210 @@
+---
+title: The Table Web Component is Now Stable - Here's What's New
+tags: [table, experimental]
+slug: /stable-table-release
+date: 2025-03-26T10:00
+---
+
+# The Table Web Component is Now Stable - Here's What's New
+
+After its half year tenure as experimental component since the relase of UI5 Web Components 2.0, allowing us to gather feedback
```

Comment:
relase => release
I usually use this tool https://www.grammarly.com/ or something similar to detect all kinds of typos and grammar mistakes before publishing